.github/workflows/shell-app-android.yml

```diff
@@ -0,0 +1,73 @@
+name: Android Shell App
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseShellAndroid:
+        description: 'type "release-shell-android" to confirm upload'
+        required: false
+  schedule:
+    - cron: '20 5 * * 2,4,6'
+  push:
+    paths:
+      - .github/workflows/shell-app-android.yml
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Get cache key of git lfs files
+        id: git-lfs
+        run: echo "::set-output name=sha256::$(git lfs ls-files | openssl dgst -sha256)"
+      - uses: actions/cache@v2
+        with:
+          path: .git/lfs
+          key: ${{ steps.git-lfs.outputs.sha256 }}
+      - run: git lfs pull
+      - name: Set up bin paths and env
+        run: |
+          echo "::add-path::$(pwd)/bin"
+          echo "::set-env name=EXPO_ROOT_DIR::$(pwd)"
+      - run: sudo apt-get install awscli
+      - name: Check that Android packages are up-to-date
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: expotools check-android-packages
+      - name: Build shell app tarball
+        run: ./buildAndroidTarballLocally.sh
```

Comment:
You didn't move the `TODO: put in fastlane?` from circleci 😃

Reply:
Yeah, I've removed it intentionally — I think it's not top priority right now and this comment has already been there for a long time (exactly as the most of our internal "for tracking" issues which are never resolved). They don't bring any value 🤷‍♂️